.github/workflows/linux.bash

```diff
@@ -9,8 +9,18 @@ CMAKE_OPTS=''
 case "$DISTRO" in
   amazonlinux:2)
     amazon-linux-extras install -y epel
-    yum install -y bison ccache cmake3 gcc-c++ flex ninja-build \
-      {libedit,mariadb,ncurses,openssl,postgresql,systemd}-devel
+
+    # The headers must be installed separately, not from openssl11-devel.
+    # That package conflicts with openssl-devel, a dependency of mariadb-devel.
+    # See: yum install {mariadb,openssl11}-devel
+    yum update -y
+    yum install -y openssl11-devel
+    mkdir /openssl11
+    cp -r /usr/include/openssl /openssl11
+    yum history undo last -y
```

Comment:
Wait what? Shouldn't this be the same EPEL package as on CentOS 7 (which can be installed side-by-side)?

Also, does that mean we might end up with a process that wants to use both OpenSSL 1.0.2 and 1.1.1? Does that even work?

Reply:
2. PoC: https://git.icinga.com/packages/icinga2/-/merge_requests/9#note_20104 (API and two IDOs work at the same time, with different OpenSSL versions 👍)